src/components/ModalAmount/styles.module.scss

```diff
@@ -0,0 +1,61 @@
+.donationsContainer {
+  height: 500px;
+  box-shadow: 0 0 0 9999px rgba(0, 0, 0, 0.5);
+}
+
+.amountContainer {
+  background-color: #f7f7f7;
+  padding: 25px 35px;
+}
+
+.selectAmtContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  margin: 15px 0px;
+}
+
+.customAmt {
+  width: 250px;
+}
+
+.nextBtn {
+  position: relative;
+  float: right;
+  right: 0px;
+  bottom: -25px;
+}
+
+.customAmt:invalid {
+  border: 1px solid red;
+}
+
+.errorMessage {
+  color: red;
+  font-size: 13px;
+}
+
+@media only screen and (max-width: 799px) {
+  .donationsContainer {
+    width: 85%;
+    height: 75vh;
```

Comment:
not super sure what vh is

Reply:
I believe it's viewport height? We should probably either avoid using it interchangeably w percentage or only use it.